mountpoint-s3-crt-sys/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "mountpoint-s3-crt-sys"
 # See `/mountpoint-s3-client/PUBLISHING_CRATES.md` to read how to publish new versions.
-version = "0.7.0"
+version = "0.7.1"
```

Comment:
This also should be 0.8.0, we included some breaking changes from the CRT e.g. the new polling interface for PutObject